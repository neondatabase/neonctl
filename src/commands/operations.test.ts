import { describe } from 'vitest';

import { test } from '../test_utils/fixtures';

describe('operations', () => {
  test('list', async ({ testCliCommand }) => {
    await testCliCommand(['operations', 'list', '--project-id', 'test']);
  });
});
