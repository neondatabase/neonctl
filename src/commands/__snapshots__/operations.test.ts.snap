// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`operations > list 1`] = `
"- id: "1"
  action: suspend
  status: finished
  created_at: 2016-01-01T00:00:00Z
- id: "2"
  action: suspend
  status: finished
  created_at: 2016-01-01T00:00:00Z
- id: "3"
  action: start
  status: finished
  created_at: 2016-01-01T00:00:00Z
"
`;
